crates/re_renderer/src/line_strip_builder.rs

```diff
@@ -43,7 +43,9 @@ impl LineStripSeriesBuilder {
                 &ctx.device,
                 &ctx.gpu_resources.buffers,
                 LineDrawData::MAX_NUM_STRIPS,
-            );
+            )
+            .unwrap_debug_or_log_error()
+            .expect("Failed to allocate picking instance id buffer"); // TODO(#3408): Should never happen but should propagate error anyways
```

Comment:
chaining `unwrap_debug_or_log_error` and `expect` makes no sense.

If you are going to panic anyway, just use `expect`

```suggestion
            .expect("Failed to allocate picking instance id buffer"); // TODO(#3408): Should never happen but should propagate error anyways
```